### DISCUSSION TITLE
Notify column resize

### DISCUSSION THREAD

**Discussion by jordibuj** [TanStack/table#2806] [Q&A] [answered]
I'd like to be notified when the columns are resized, so I can store their widths in the user's settings.
I found that in version 6 there used to be an onResizedChange callback, which looked like what I need, but I'm on version 7 and I can't find anything similar... is it still possible to be notified when a column is resized?

**Comment by wildcat-cs** [comment]
I'm hoping you get an answer to this as well.  I want to know when the columns are resize to recalculate row heights on a virtual grid but I think it's the same question.

**Comment by gargroh** [comment, marked as answer]
You can watch for `columnResizing` state change(https://github.com/tannerlinsley/react-table/blob/master/src/plugin-hooks/useResizeColumns.js#L144) using useEffect
